drivers/phoenixcontact_modbus.c

```diff
@@ -101,7 +101,13 @@ upsdrv_info_t upsdrv_info = {
 void upsdrv_initinfo(void)
 {
 	uint16_t tab_reg[4];
-	uint64_t PartNumber;
+	uint64_t PartNumber = 0;
+	size_t i;
+
+	for (i = 0; i < (sizeof(tab_reg) / sizeof(tab_reg[0])); i++)
+	{
+		tab_reg[i] = 0;
+	}
```

Comment:
Thinking of it, some `memset()` to `0` would have been more idiomatic :) But, "oh well".